server/app/routes/product/index.js

```diff
@@ -27,8 +27,11 @@ router.param('productId', function(req, res, next, productId) {
 
 //Get all products
 router.get('/', function(req, res, next) {
-    var query = req.query.name ? {name: {$iLike: '%' + req.query.name + '%'}} : {}
-    Product.findAll({ where: query })
+    var query = req.query.name ? { name: { $iLike: '%' + req.query.name + '%' } } : {}
+    Product.findAll({
+            where: query,
+            include: [{model: Tag}]
```

Comment:
merge conflict here. should copy the code from master right now into this file and make this 

```
include: [{model: Tag}]

```

and then try repushing and pull request